parsers/AR.py

```diff
@@ -143,8 +158,50 @@ def fetch_exchange(
     logger: Logger = getLogger(__name__),
 ) -> dict:
     """Requests the last known power exchange (in MW) between two zones."""
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+
+    sorted_zone_keys = sorted([zone_key1, zone_key2])
+    sorted_codes = "->".join(sorted_zone_keys)
+    flow: Union[float, None] = None
+    target_datetime = arrow.now(tz="America/Argentina/Buenos_Aires").datetime
```

Comment:
This is actally never used an therefore not needed.

Reply:
I added this in the failure case where the exchange isn't returned in the API response (i.e. `if properties["nombre"] == exchange_name` never evaluates to true), in which case the flow returned would be `None`.

Reply:
If the exchange is not returned we should raise a `ParserException` instead of returning None.
I was mostly referring to assigning a date to `target_datetime` though that is never used later (and not needed as we get the returned datetime from the API).